apps/web/playwright/onboarding.e2e.ts

```diff
@@ -24,10 +24,6 @@ test.describe("Onboarding", () => {
       await expect(page.locator('text="Next Step"')).toBeVisible(); // Fix race condition
 
       await test.step("step 1 - User Settings", async () => {
-        // Check required fields
-        await page.locator("button[type=submit]").click();
-        await expect(page.locator("data-testid=required")).toBeVisible();
```

Comment:
not needed

Reply:
Why not?

Reply:
ah I meant to revert this, reverted now!